simple_ldap_user/simple_ldap_user.admin.inc

```diff
@@ -297,14 +297,14 @@ function simple_ldap_user_import($form, &$form_state) {
   // Only selects new users
   
   $new_names = $users;
-  $sql="select name from {users} where name= :name";
-  foreach($new_names as $item){
-     $result=db_query($sql, array(':name'=>$item));
-     if($result)
-     {
-        while($row=$result->fetchAssoc()){
-            #Filter users fetched from ldap database and mysql database
-            unset($new_names[strtoupper($row['name'])]);
+  $sql = "select name from {users} where name= :name";
+  foreach ($new_names as $item){
+     $result = db_query($sql, array(':name'=>$item));
+     if ($result) {
+        while ($row=$result->fetchAssoc()) {
+            // Filter users fetched from ldap database and mysql database
+
+            unset ($new_names[strtoupper($row['name'])]);
```

Comment:
This module does adhere to the code standards requirement of two spaces per indent.  Let's keep our changes in line with that.  Sorry for not catching that earlier.